src/types.ts

```diff
@@ -342,7 +470,7 @@ export interface UpdateQueue {
     queue: UpdateQueueQueue;
 }
 
-interface UpdateQueues {
+interface UpdateQueuesCommon {
```

Comment:
I guess this should not contain bakerStakeThreshold, as it only appears in V0

Reply:
I guess it is not the same as the authorizations

Reply:
Good catch!

Reply:
Had a talk with @td202 , and the authorizations prop is kept the same, while the update queue is renamed. Will add a comment to explain on Authorizations interface.